src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -2228,3 +2227,52 @@ commuting with the contravariant lifts.
              x y f (g b))))
 
 ```
+
+## Discrete fibers
+
+The fibers of a covariant fibration over a Segal type are discrete types.
+
+```rzk title="RS17, Proposition 8.18"
+#def is-discrete-is-covariant-segal
+  ( A : U)
+  ( is-segal-A : is-segal A)
+  ( C : A → U)
+  ( is-cov-C : is-covariant A C)
+  ( x : A)
+  : is-discrete (C x)
+  :=
+    ( \ u v → total-equiv-family-of-equiv
+      ( C x)
+      ( \ v' → (u = v'))
+      ( hom (C x) u)
+      ( hom-eq (C x) u)
+      ( is-equiv-are-contr
+        ( Σ (y : (C x)) , u = y)
+        ( Σ (y : (C x)) , hom (C x) u y)
+        ( is-contr-based-paths (C x) u)
+        ( is-cov-C x x (id-hom A x) u)
+        ( total-map
+          ( C x)
+          (\ v' → u = v')
```

Comment:
A nitpick but there should be an indentation before the "\" so it lines up with the "C" on the previous line.